src/results/electric_storage.jl

```diff
@@ -64,6 +64,7 @@ function add_electric_storage_results(m::JuMP.AbstractModel, p::REoptInputs, d::
                 r["replacement_month"] = Int(value(
                     sum(mth * m[:bmth][mth] for mth in 1:p.s.financial.analysis_years*12)
                 ))
+                r["residual_value"] = value(m[:salv_value])
             end
```

Comment:
Should we include alternative values for these if the strategy is `"augmentation"` or `p.s.storage.attr[b].model_degradation` is False?  Or do we only output these for the degradation model?  Thinking of how this should be implemented in the API endpoint.

Reply:
Good question. Currently these fields are only in the output if degradation is true. These fields can be handled in the API by creating nullable and blankable output fields so API would catch them if REopt were to return them.